.github/actions/setup/action.yml

```diff
@@ -25,7 +25,9 @@ runs:
         sudo add-apt-repository ppa:ubuntu-toolchain-r/test
 
         sudo apt-get update
-        sudo apt-get install autoconf autoconf-archive automake build-essential cmake libavcodec-dev fonts-liberation2 zip curl tar ccache clang-18 clang++-18 lld-18 gcc-13 g++-13 libstdc++-13-dev ninja-build unzip qt6-base-dev qt6-tools-dev-tools libqt6svg6-dev qt6-multimedia-dev libgl1-mesa-dev libpulse-dev libssl-dev libegl1-mesa-dev
+        sudo apt-get install autoconf autoconf-archive automake build-essential cmake libavcodec-dev fonts-liberation2 zip curl tar ccache clang-18 clang++-18 lld-18 gcc-13 g++-13 libstdc++-13-dev \
+            ninja-build unzip qt6-base-dev qt6-tools-dev-tools libqt6svg6-dev qt6-multimedia-dev libgl1-mesa-dev libpulse-dev libssl-dev libegl1-mesa-dev \
+            libx11-dev libxrandr-dev
```

Comment:
Not happy about these new deps. Feels gross to need both x11 and Wayland headers, regardless of what your distro is.

But, without pulling vulkan-headers and vulkan from vcpkg, developers who have Vulkan development libs on their system run into header-mixing problems anyway. Since skia will be pulling vulkan from vcpkg, and LibCore would be pulling it from your system.

:T